install_config/cluster_metrics.adoc

```diff
@@ -544,3 +544,19 @@ For more information about Heapster and how to access its APIs, please refer the
 link:https://github.com/kubernetes/heapster/[Heapster] project.
 
 endif::[]
+
+== Cleanup
+You can remove your metrics deployment by doing:
+[options="nowrap"]
```

Comment:
You can take this line out because `:prewrap!:` is in use in the topic metadata.